features/checkout/seeing_order_summary/seeing_addresses_on_order_summary_page.feature

```diff
@@ -0,0 +1,33 @@
+@checkout
+Feature: Seeing order addresses on order summary page
+    In order to be certain about shipping and billing address
+    As a Customer
+    I want to be able to see addresses on the order summary page
+
+    Background:
+        Given the store operates on a single channel in "France"
+        And the store has a product "Lannister Coat" priced at "$19.99"
+        And the store ships everywhere for free
+        And the store allows paying offline
+        And I am a logged in customer
+
+    @ui
+    Scenario: Seeing the same shipping and billing address on order summary
+        Given I have product "Lannister Coat" in the cart
+        When I specified the shipping address as "Ankh Morpork", "Frost Alley", "90210", "France" for "Jon Snow"
+        And I complete order with "Free" shipping method and "Offline" payment
+        Then I should be on the checkout summary step
+        And I should see this shipping address as shipping and billing address
```

Comment:
Another solution for different billing and shipping address is to keep the address in SharedStorage by the fullname of recipient and then do something like:

```
When I specify the shipping address as "Ankh Morpork", "Frost Alley", "90210", "France" for "Jon Snow"
And I specify the billing address as "Ankh Morpork", "Frost Alley", "90210", "France" for "Eddard Stark"
And I complete the addressing step
And I proceed with "Free" shipping method and "Offline" payment
Then I should be on the checkout summary step
And my order's shipping address should be to "Jon Snow"
But my billing should be to "Eddard Stark"
```

We can only check the firstName and lastName on the order or like I said: store addresses per fullName in shared storage and hide the full comparision in the context.

Another side note: Step "I choose different billing address" should not be needed. The action of clicking the checkbox (which is a specific UI implementation) should be a part of the "I specify billing address" page implementation.